pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyConnection.java

```diff
@@ -543,6 +577,35 @@ protected void handleAuthResponse(CommandAuthResponse authResponse) {
         }
     }
 
+    private Void requestRefreshClientAuthData() {
+        if (!service.getConfiguration().isForwardAuthorizationCredentials()) {
```

Comment:
I'm not sure if we have to check this again. because it looks like already checked at `ProxyClientCnx`. but it's fine.

Reply:
Ok, I can remove this check.